[2025-06-07T12:49:26.181+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-07T12:49:26.192+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: curry_medallion_etl.ensure_datalake_bucket_exists manual__2025-06-07T12:49:25.941296+00:00 [queued]>
[2025-06-07T12:49:26.196+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: curry_medallion_etl.ensure_datalake_bucket_exists manual__2025-06-07T12:49:25.941296+00:00 [queued]>
[2025-06-07T12:49:26.197+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-06-07T12:49:26.204+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): ensure_datalake_bucket_exists> on 2025-06-07 12:49:25.941296+00:00
[2025-06-07T12:49:26.208+0000] {standard_task_runner.py:72} INFO - Started process 1994 to run task
[2025-06-07T12:49:26.210+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'curry_medallion_etl', 'ensure_datalake_bucket_exists', 'manual__2025-06-07T12:49:25.941296+00:00', '--job-id', '76', '--raw', '--subdir', 'DAGS_FOLDER/medallion_sales_etl_dag.py', '--cfg-path', '/tmp/tmp_3bknpnk']
[2025-06-07T12:49:26.212+0000] {standard_task_runner.py:105} INFO - Job 76: Subtask ensure_datalake_bucket_exists
[2025-06-07T12:49:26.249+0000] {task_command.py:467} INFO - Running <TaskInstance: curry_medallion_etl.ensure_datalake_bucket_exists manual__2025-06-07T12:49:25.941296+00:00 [running]> on host 5ebb95484de3
[2025-06-07T12:49:26.305+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='qdinh18' AIRFLOW_CTX_DAG_ID='curry_medallion_etl' AIRFLOW_CTX_TASK_ID='ensure_datalake_bucket_exists' AIRFLOW_CTX_EXECUTION_DATE='2025-06-07T12:49:25.941296+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-07T12:49:25.941296+00:00'
[2025-06-07T12:49:26.306+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-07T12:49:26.319+0000] {logging_mixin.py:190} INFO - Bucket 'mybucket' already exists.
[2025-06-07T12:49:26.320+0000] {python.py:240} INFO - Done. Returned value was: True
[2025-06-07T12:49:26.337+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-07T12:49:26.338+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=curry_medallion_etl, task_id=ensure_datalake_bucket_exists, run_id=manual__2025-06-07T12:49:25.941296+00:00, execution_date=20250607T124925, start_date=20250607T124926, end_date=20250607T124926
[2025-06-07T12:49:26.352+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-07T12:49:26.368+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-07T12:49:26.369+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
